e2e/log.go

```diff
@@ -0,0 +1,55 @@
+package e2e
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+
+	collector "github.com/coreos/ktestutil/log-collector"
+)
+
+var cr *collector.Collector
+
+func startLogging(keypath, dstDir string) error {
+	cr = collector.New(&collector.Config{
+		K8sClient:     client,
+		Namespace:     namespace,
+		RemoteKeyFile: keypath,
+	})
+	if err := cr.Start(); err != nil {
+		return err
+	}
+
+	dirAbsPath, err := filepath.Abs(dstDir)
+	if err != nil {
+		return err
+	}
+	os.Mkdir(dirAbsPath, 0777)
+	if err := cr.SetOutputToLocal(dirAbsPath); err != nil {
+		return err
+	}
+	return nil
+}
+
+func collectLogs() error {
+	pods, err := cr.CollectPodLogs("*")
+	if err != nil {
+		return err
+	}
+	log.Println("pods", pods)
```

Comment:
Sure I can change it. Options:
* `[count] log files collected for pods`
* remove the Println altogether
@yifan-gu